torch/distributed/nn/api/remote_module.py

```diff
@@ -50,6 +57,16 @@ def __init__(
         The arguments of ``forward_async`` and ``forward`` are the same as
         the ``forward`` method of the module returned by the ``module_cls``.
 
+        Since all the other methods except ``forward`` and ``forward_async``
+        directly inherit from ``nn.Module``,
+        these methods are only applied to the caller on the local side,
+        instead of the user-provided ``module_cls`` that will be executed on the remote node.
+        Therefore, normally these methods should not be used, and the usage will throw exceptions.
+        Particularly, to create a hybrid model, typically the local modules should be
+        in parallel with remote modules, rather than as submodules of any remote module.
```

Comment:
I'm not sure I followed this comment completely, what do you mean by the modules should be in "parallel" to remote modules?

Reply:
I mean the relationship of any local modules with a remote module in a hybrid model should be the peers, rather than containment by calling add_module method.

Example:
class HybridModel(nn.Module):
    def __init__(self):
        nn.Module.__init__(self)
        self.remote_embedding = RemoteModule(
            nn.Embedding,
            num_embeddings=1024,
            embedding_dim=4,
            on='embedding_server')
        self.local_linear = nn.Linear(
            in_features=4,
            out_features=2,
        )

Updated the comment.